.ci/jobs/beats.yml

```
@@ -17,7 +17,7 @@
         discover-pr-forks-strategy: 'merge-current'
         discover-pr-forks-trust: 'permission'
         discover-pr-origin: 'merge-current'
-        head-filter-regex: '(master|6\.[89]|7\.[x789]|7\.1\d|8\.\d+|PR-.*|v\d+\.\d+\.\d+)'
+        head-filter-regex: '(master|7\.16|8\.\d+|PR-.*|v\d+\.\d+\.\d+)'
```

If `6.8` branch is still an active release branch, then we need it :)

```suggestion
        head-filter-regex: '(master|6\.[89]|7\.16|8\.\d+|PR-.*|v\d+\.\d+\.\d+)'
```

---

It is not active. (Or at least it shouldn't be.) It was [EOL in November of 2020](https://www.elastic.co/support/eol).

---

Uhm, wait, that' over a year ago, but they still release patches https://github.com/elastic/beats/releases/tag/v6.8.20

---

Under the `Maintenance policy`

> This last minor will be maintained until the release of the second subsequent major version. For example, Elasticsearch 6.8.x will be maintained until the GA release of Elasticsearch 8.0.0. At the release of Elasticsearch 8.0.0, we will continue to maintain the last 7.x series, and begin maintaining the 8.0.x minor series, then 8.1.x series, then 8.2.x series of minor releases.


Did we release `GA` for 8.0.0 yet?

---

Well...that is very interesting.

@andresrc or @jlind23 can you help us understand this?

---

@v1v Ah, OK. I find that language extremely confusing since it sounds like we're saying that 6.8 is both EOL and maintained. Regardless, I think you're right here and we should not remove 6.8 from this list. I'll update the PR now. Thanks!